src/drivers/mct/bld/namelist_files/namelist_definition_drv_flds.xml

```diff
@@ -0,0 +1,131 @@
+    List of possible MEGAN compounds to use
+    (the list used by the simulation is on the megan_factors_file as the Comp_Name)
+  </entry>
+
+  <!-- ========================================================================================  -->
+  <!-- drydep Namelists                                                                          -->
+  <!-- ========================================================================================  -->
+
+  <entry id="drydep_method"
+	 type="char*16"
+	 category="dry-deposition"
+	 group="drydep_inparm"
+	 valid_values="xactive_lnd,xactive_atm,table">
+    Where dry deposition is calculated (from land, atmosphere, or from a table)
+    This specifies the method used to calculate dry
+    deposition velocities of gas-phase chemical species.  The available methods
+    are:
+    'table'       - prescribed method in CAM
+    'xactive_atm' - interactive method in CAM
+    'xactive_lnd' - interactive method in CLM
+  </entry>
+
+<entry id="drydep_list"
+       type="char*32(300)"
+       category="dry-deposition"
+       group="drydep_inparm"
+       valid_values="">
+    List of species that undergo dry deposition.
+  </entry>
+
+
+  <!-- ========================================================================================  -->
+  <!-- Fire emissions fluxes                                                                     -->
+  <!-- ========================================================================================  -->
+
+  <entry id="fire_emis_factors_file"
+	 type="char*256"
+	 input_pathname="abs"
+	 category="Fire_emissions"
+	 group="fire_emis_nl"
+	 valid_values="" >
+    File containing fire emissions factors.
+    Default: none
+  </entry>
+
+  <entry id="fire_emis_specifier"
+	 type="char*1024(100)"
+	 category="Fire_emissions"
+	 group="fire_emis_nl"
+	 valid_values="" >
+    Fire emissions specifier.
+    Default: none
+  </entry>
+
+  <entry id="fire_emis_elevated"
+	 type="logical"
+	 category="Fire_emissions"
+	 group="fire_emis_nl"
+	 valid_values="" >
+    If ture fire emissions are input into atmosphere as elevated forcings.
+    Otherwise they are treated as surface emissions.
+    Default: TRUE
+  </entry>
+
+  <!-- ========================================================================================  -->
+  <!-- CARMA fields                                                                              -->
+  <!-- ========================================================================================  -->
+
+  <entry id="carma_fields" type="char*256" category="carma"
+	 group="carma_inparm" valid_values="" >
+    List of fluxes needed by the CARMA model, from CLM to CAM.
+  </entry>
+
+</namelist_definition>
```

Comment:
The above has the wrong schema for xml driver cime_config/ files. These entries are already in namelist_definition_drv_flds.xml and should not be replicated here. If ALM needs these fields - they should be in the ALM bld/ directory (and the schema above is consistent with having them in the ALM bld/ directory) and not the driver cime_config/ directory.